ext/hal/nordic/nrfx_config_nrf52811.h

```diff
@@ -0,0 +1,2448 @@
+/**
```

Comment:
There are a couple of things that are missing or need to be corrected in this file, like NVMC and TEMP sections, CLOCK_CONFIG_LF_SRC settings, and UART enabling. I'd recommend to take `nrfx_config_nrf52810.h` from the latest master as a starting point and only add SPI[M|S]1 related entries to it achieve the desired consistency.

Reply:
done